hbase-server/src/main/java/org/apache/hadoop/hbase/tool/Canary.java

```diff
@@ -1199,6 +1216,28 @@ private void monitorRegionServers(Map<String, List<HRegionInfo>> rsAndRMap) {
       return rsAndRMap;
     }
 
+    private Set<String> getDrainingServers() {
+      Configuration conf = this.connection.getConfiguration();
+      String baseZNode = conf.get(HConstants.ZOOKEEPER_ZNODE_PARENT,
+        HConstants.DEFAULT_ZOOKEEPER_ZNODE_PARENT);
+      String drainingZNode = ZKUtil.joinZNode(baseZNode,
+        conf.get("zookeeper.znode.draining.rs", "draining"));
+      Set<String> drainingServers = new HashSet<>();
+      try {
+        ZooKeeperWatcher zkw = new ZooKeeperWatcher(conf, CANARY_TOOL + connection.toString(),
+          new LoggingAbortable());
```

Comment:
I think because this is being used from the client context, we can just pass in null instead of creating an abortable.